cmd/opampsupervisor/e2e_test.go

```diff
@@ -133,7 +133,9 @@ func newOpAMPServer(t *testing.T, connectingCallback onConnectingFuncFactory, ca
 
 func newSupervisor(t *testing.T, configType string, extraConfigData map[string]string) *supervisor.Supervisor {
 	cfgFile := getSupervisorConfig(t, configType, extraConfigData)
-	s, err := supervisor.NewSupervisor(zap.NewNop(), cfgFile.Name())
+	logger, err := zap.NewDevelopment()
```

Comment:
How noisy is this when `go test` is run without the `-v` flag?

Reply:
It was noisy but not too much. I can revert this change if you think the logs are too many. It was helpful for me to debug the failed tests.